neurovault/apps/statmaps/urls.py

```diff
@@ -49,6 +49,9 @@
     url(r'^collections/(?P<cid>\d+|[A-Z]{8})/delete$',
         delete_collection,
         name='delete_collection'),
+    url(r'^collections/(?P<cid>\d+|[A-Z]{8})/download$',
```

Comment:
I wonder if it would make sense to change this to reflect the fact that the response is a zip file.  For example replace "download" with `"%s.zip" % collection.name`?